widget/discrepancy-note/dn-widget.js

```diff
@@ -531,38 +531,39 @@ class Comment extends Widget {
     }
 
     /**
-     * Sets users, currentUser, and usersOptionsHtml global variables (once for all dn widgets);
-     * 
-     * @param {boolean=} readOnly 
+     * Sets users and currentUser global variables (once for all dn widgets);
      */
-    _setUserOptions( readOnly ) {
-        if ( !usersOptionsHtml ) {
-            const disabled = readOnly ? 'disabled' : '';
-            const defaultAssignee = this.defaultAssignee;
+    _setUsers() {
+        if ( !currentUser || !users ) {
```

Comment:
If for some reason OC does not mark a user as current, it will re-evaluate the list of users for every widget. I think it's better to just do `if( !users )` here

Reply:
code updated